### ISSUE TITLE
request:  stop `flutterfire configure` from repeatedly logging out the same line when creating a new project

### ISSUE THREAD

**Issue by russellwheatley** [invertase/flutterfire_cli#147]
### Is there an existing feature request for this?

- [X] I have searched the existing issues.

### Description

<img width="925" alt="screenshot_2023-02-10_at_16 23 27" src="https://user-images.githubusercontent.com/16018629/218129773-6c1f2c98-e0f5-4de9-9d96-1827c5e8985a.png">

Perhaps change to a loading symbol so users know it is in progress rather than printing out the same line.

### Reasoning

Improves user experience.

**Comment by exaby73** [comment]
Hey @russellwheatley. I was trying this command on my machine, and it does not look like I can reproduce. I ran the command `flutterfire configure` in the VsCode terminal and it seemed to work as expected. Is it possible that this was fixed in `interact` which is what `flutterfire_cli` uses for the spinner? Or is this specific to a specific terminal? I tried Alacritty on Linux and Mac and the MacOS default terminal as well and I could not reproduce it
